build.sbt

```diff
@@ -77,8 +77,8 @@ Global / resolvers +=
   "scala-integration".at("https://scala-ci.typesafe.com/artifactory/scala-integration/")
 
 val commonJsSettings = Seq(
-  crossScalaVersions := List(LatestScala213, LatestScala212),
```

Comment:
should we add a `val LatestScala213ForJS`, perhaps? :)

Reply:
Done